GradedModulePresentationsForCAP/gap/GradedModulePresentations.gi

```diff
@@ -574,10 +582,20 @@ InstallGlobalFunction( ADD_GRADED_LIFT,
         lift := Lift( UnderlyingPresentationMorphism( alpha ), UnderlyingPresentationMorphism( beta ) );
         
         if lift = fail then
+        
             return fail;
+        
         fi;
         
-        return GradedPresentationMorphism( Source( alpha ), lift, Source( beta ) );
+        lift := GradedPresentationMorphism( Source( alpha ), lift, Source( beta ) );
+        
+        if not IsWellDefined( lift ) then
+          
+          Error( "An output of the Lift method is not well-defined!" );
```

Comment:
I think this is not the right way to do this, for two reasons:

1. Primitive implementations should not secretly depend on other primitives, like `IsWellDefined` here. But this is not too bad.

2. `Lift` can return `fail`, and I would think it is appropriate to return `fail` here.

Reply:
This change in `Lift` and `Colift` was supposed to be temporary and is outdated by #262.

Reply:
I just approved #262. If you are fine with it, we can merge it and then update this PR and merge it, too?

Reply:
Yes.